src/components/VoteScreen.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import CandidateStatus from './CandidateStatus.js';
+import LoginInput from './LoginInput';
+import { useCookies } from 'react-cookie';
+
+const candiadateURL='http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/candidates'
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
+  const [cookies, removeCookie] = useCookies(['token']);
+
+  useEffect(() => {
+    const getCandidateList = async () => {
+      try {
+        const response = await axios.get(
+          candiadateURL
+        );
+        setCandidates(response.data);
+       
+      } catch (e) {
+        console.log(e);
+      }
+    };
+    getCandidateList();
+  }, [candidates]);
+
+  if (!candidates) {
+    return null;
+  }
+
+  candidates.sort((a, b) => b.voteCount - a.voteCount);
+
+  return (
+    <Wrapper>
+      <Intro>CEOS 프론트앤드 13기 개발팀장을 뽑자 🎉</Intro>
+      <LoginInput></LoginInput>
+      {candidates.map((candidate, index) => {
+        return (
+          <CandidateStatus
+            key={candidate.id}
+            candidate={candidate}
+            order={index + 1}
+            cookies={cookies}
+          ></CandidateStatus>
```

Comment:
`CandidateRow`라는 이름도 괜찮을 것 같네용
```suggestion
          <CandidateRow
            key={candidate.id}
            candidate={candidate}
            order={index + 1}
            cookies={cookies}
          ></CandidateRow>
```